src/averageAges.js

```diff
@@ -14,14 +14,23 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+const MALE = 'f';
+const FEMALE = 'f';
+const ONE_CENTURY = 1000;
```

Comment:
The constants MALE and FEMALE are incorrectly assigned values. MALE should be 'm' and FEMALE should be 'f'. Please correct the assignments.